back-end/__tests__/app.test.js

```diff
@@ -141,3 +141,23 @@ describe("/api/league_tables", () => {
       });
   });
 });
+
+describe("/api/venues", () => {
+  test("GET 200: Should return an array of venue objects", () => {
+    return request(app)
+      .get("/api/venues")
+      .expect(200)
+      .then(({ body }) => {
+        const { venues } = body;
+        expect(venues).toHaveLength(5);
```

Comment:
Hiya Mior, looks like there is 10 in the test data, but this only checks for 5, can we make this match please otherwise looks good!

Reply:
Thanks @Alfie-F , as discussed on Slack, the 5 is due to the now trimmed test file that reduces the number of venues to 5.